test-browser/tests/units/testRecorder.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         .clickFunction('getFromLib - call')
         .waitForElementPresent('div[class^="contractActionsContainer"] div[class^="value"] ul')
         .perform((client, done) => {
-          contractHelper.verifyCallReturnValue(browser, '0x35ef07393b57464e93deb59175ff72e6499450cf', ['0: uint256: 1', '0: uint256: 3456', '0: address: 0x35ef07393b57464e93deb59175ff72e6499450cf'], () => {
+          contractHelper.verifyCallReturnValue(browser, '0x35ef07393b57464e93deb59175ff72e6499450cf', ['0: uint256: 1', '0: uint256: 3456', '0: address: 0x35eF07393b57464e93dEB59175fF72E6499450cF'], () => {
```

Comment:
This part I don't quite understand - why are some letter capitalized?

Reply:
the old decoder would just return basic hex number, the new one take care of returning a valid checksum address

Reply:
the function is here : https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/index.md

Reply:
Cool. Very helpful.